package.linux.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
why do we need to have a separate package.json file for linux?

Reply:
Because I developed this project on a windows machine and some commands in package file are windows based e.g. copy file command. So after completing my work I spinned up a linux machine and created another working package file for reviewer(assuming that reviewer would have linux based system).